templates/default/sql/dml/3.1.0.sql.erb

```diff
@@ -8,3 +8,6 @@ DELETE FROM `hopsworks`.`variables` WHERE `id`='dela_hopsworks_public_port';
 DELETE FROM `hopsworks`.`variables` WHERE `id`='hops_site_host';
 DELETE FROM `hopsworks`.`variables` WHERE `id`='hopssite_heartbeat_interval';
 DELETE FROM `hopsworks`.`variables` WHERE `id`='hops_site_endpoint';
+
+
+REPLACE INTO `hopsworks`.`variables`(`id`, `value`) VALUES('read_only_git_repositories', '<%= node['hopsworks']['read_only_git_repositories'] %>');
```

Comment:
```suggestion
REPLACE INTO `hopsworks`.`variables`(`id`, `value`) VALUES('read_only_git_repositories', '<%= node['hopsworks']['read_only_git_repositories'] %>', 1);
```

Reply:
Right now the visibility is set such that only the admin can see the variable. We need to set it to 1 so that also users can access this variable.